src/System.Buffers/src/System/Buffers/ArrayPool.cs

```diff
@@ -63,6 +62,12 @@ public static ArrayPool<T> Create(int maxArrayLength, int numberOfArrays)
         }
 
         /// <summary>
+        /// Gets the EventSource responsible for tracing events in the ArrayPool used for debugging
+        /// and diagnostics around Pool usage.
+        /// </summary>
+        public abstract EventSource TraceEventSource { get; }
```

Comment:
I think it has been mentioned somewhere else, but I really would like to avoid exposing any APIs related to tracing.